bluesky/traffic/route.py

```diff
@@ -138,73 +155,71 @@ def addwptStack(self, idx, *args):  # args: all arguments of addwpt
                 if self.wpname[i].count("/") >0:
 #                    print (self.wpname[i])
                     rwyrteidx = i
-                i = i + 1
+                i += 1
 
             # Only TAKEOFF is specified wihtou a waypoint/runway
-            if len(args) == 1 or args[1] == "" or args[1] is None:
+            if len(args) == 1 or not args[1]:
                 # No runway given: use first in route or current position
 
                 # print ("rwyrteidx =",rwyrteidx)
                 # We find a runway in the route, so use it
                 if rwyrteidx>0:
-                    rwlat   = self.wplat[rwyrteidx]
-                    rwlon   = self.wplon[rwyrteidx]
-                    aptidx  = bs.navdb.getapinear(rwlat,rwlon)
+                    rwylat   = self.wplat[rwyrteidx]
+                    rwylon   = self.wplon[rwyrteidx]
+                    aptidx  = bs.navdb.getapinear(rwylat,rwylon)
                     aptname = bs.navdb.aptname[aptidx]
 
-                    if self.wpname[rwyrteidx].count("/")>1:

```

Comment:
the count on '/' need to be >0, otherwise you will end up with something like `EHAM/RW06`  as the runway.  The `if` statement is then redundant as this is always true (re, `rwyrteidx>0`, line 149 in original code)